From 81ab4a29f294aa875f636bef220632bf41b3a79e Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 11 May 2022 18:05:54 +0200
Subject: [PATCH 209/841] feat(stm32mp2-fdts): add package otp in STM32MP251

Add the OTP used to select the package for STM32MP25x.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 515d12ee991378ba267533794279219b6331fad4 feat(stm32mp2-fdts): add package otp in STM32MP251]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I6eda2e2befdeb5040a6324931841c17ef031c3f7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/251865
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 fdts/stm32mp251.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 48cd7bd9b..5a890e86c 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -297,6 +297,9 @@
 			nand2_otp: otp20@50 {
 				reg = <0x50 0x4>;
 			};
+			package_otp@1e8 {
+				reg = <0x1e8 0x1>;
+			};
 		};
 
 		iwdg1: watchdog@44010000 {
-- 
2.47.2

